backend/src/main/java/gov/cdc/usds/simplereport/logging/PatientExperienceLoggingInterceptor.java

```diff
@@ -20,12 +20,12 @@ public class PatientExperienceLoggingInterceptor implements HandlerInterceptor {
       LoggerFactory.getLogger(PatientExperienceLoggingInterceptor.class);
 
   @Override
-  public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler)
-      throws Exception {
+  public boolean preHandle(
+      HttpServletRequest request, HttpServletResponse response, Object handler) {
     log.trace(
         "Pre-handling a method={} request for uri={} using handler={}",
         request.getMethod(),
-        request.getRequestURI(),
+        sanitizeRequestURI(request.getRequestURI()),
```

Comment:
⚠️ This was previously unsanitized log input - users could theoretically inject special characters, etc into our logs by hitting a specific request URI. 

See here for details: https://github.com/CDCgov/prime-simplereport/security/code-scanning/72